modules/veteran_confirmation/VETERAN_CONFIRMATION.yml

```diff
@@ -51,24 +64,40 @@ paths:
                 properties:
                   data:
                     $ref: '#/components/schemas/VeteranStatusConfirmation'
-        '502':
-          description: eMIS failed to respond or responded in a way we cannot handle. 
+        '503':
```

Comment:
The Facilities OAS has good examples of additional error statuses that can be added: https://github.com/department-of-veterans-affairs/vets-api/blob/cfd79ba48a17545f5b340217a7fa0e038a8b394f/modules/va_facilities/README.yml#L131